Integrations/build.gradle

```diff
@@ -87,7 +87,9 @@ task deephavenPythonDockerfile(type: Dockerfile) {
     copyFile(new Dockerfile.CopyFile('/usr/src/app/dist', '.').withStage('deephaven-jpy-wheel'))
     copyFile(new Dockerfile.CopyFile('/usr/src/app/dist', '.').withStage('deephaven-wheel'))
 
-    runCommand('''set -eux;\\
```

Comment:
@devinrsmith There is probably a better place/way to do this, let me know what is best here.

Reply:
This is an OK place to do it. I'm not a fan of adding universal dependencies to our base docker images; but liblzo2-2 is very small. The better place is *probably* py/jpy/Dockerfile under the runtime_reqs heading (deephaven/java-and-python). (It is a little weird that this file is currently under the jpy heading - we could refactor in the future, but I'm not too worried about that part.)

At the very least, we need to add a comment about why we are adding liblzo2-2.